tests/suites/0_stateless/02_function/02_0003_function_database.sql

```diff
@@ -1,2 +1,6 @@
-SELECT database(), currentDatabase();
+SELECT database(), currentDatabase(), current_database();
 
+set enable_planner_v2 = 1;
+SELECT database(), currentDatabase(), current_database();
```

Comment:
better to add this file: tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql